django/contrib/admin/models.py

```diff
@@ -73,14 +73,14 @@ def __repr__(self):
 
     def __str__(self):
         if self.is_addition():
-            return gettext('Added "%(object)s".') % {'object': self.object_repr}
+            return gettext('Added “%(object)s”.') % {'object': self.object_repr}
         elif self.is_change():
-            return gettext('Changed "%(object)s" - %(changes)s') % {
+            return gettext('Changed “%(object)s” - %(changes)s') % {
                 'object': self.object_repr,
                 'changes': self.get_change_message(),
             }
         elif self.is_deletion():
-            return gettext('Deleted "%(object)s."') % {'object': self.object_repr}
+            return gettext('Deleted “%(object)s.”') % {'object': self.object_repr}
```

Comment:
Should this period be moved outside the quotes?

Reply:
This is inconsistent with the "Added" case a few lines above as well as several other lines in the diff. Should we normalize these sentences to period inside quotes?

Reply:
In American English, the punctuation usually goes inside the quote.

Reply:
Does that mean we should normalize to this style as part of this change?

Reply:
Seems like it.

But it feels strange to me to have `Deleted "Bob."` instead of `Deleted "Bob".` - or am I missing something here?

Reply:
I agree that it _feels_ strange. I'll go with whatever is understood to be the most correct grammar style for American English.